AwooChat.py

```diff
@@ -29,84 +29,76 @@ def titleWatchdog(msg):
         for i in AwooUtils.getChatAdmins(groupId):
             bot.sendMessage(i, "[ADM] A Group Name has been changed by " + offendingUsername + ". New name: " + offendingTitle)
 
+def parsePokemon(source):
+    parsedPokemon = {}
+    s = json.load(urllib2.urlopen(source))
+    pokemonList = s["pokemons"]
+
+    for i in pokemonList:
+        id = i['pokemon_id']
+        lat = i['latitude']
+        lon = i['longitude']
+        enc_id = i['encounter_id']
+        despawn = datetime.fromtimestamp(i['disappear_time'] / 1000).strftime('%H:%M:%S')
+
+        if (enc_id not in pokemonWatchdogSeen):
+            parsedPokemon[id] = parsedPokemon.get(id, [])
+            parsedPokemon[id].append({'lat': lat, 'lon': lon, 'despawn': despawn})
+            pokemonWatchdogSeen.append(enc_id)
+
 def pokemonWatchdog():
     global pokemonWatchdogSeen
-    p = prefs
```

Comment:
This was the only function that used `p = prefs` and it was not consistently used so I changed back to using `prefs` for readability. We can use `p` instead of `prefs` but we should do that everywhere or nowhere.